<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneBook</class>
 <widget class="QMainWindow" name="PhoneBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhoneBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: #ffffff;
}</string>
   </property>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>551</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Umpush</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Phone Book</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_contact">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 3px solid #6c757d;
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_contact">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 3px solid #6c757d;
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>591</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </column>
   </widget>
   <zorder>title</zorder>
   <zorder>picture</zorder>
   <zorder>add_contact</zorder>
   <zorder>delete_contact</zorder>
   <zorder>tableWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
